charts/pg-db/values.yaml

```diff
@@ -311,6 +311,7 @@ proxy:
 #          pool_mode: transaction
 
 backups:
+  trackLatestRestorableTime: true
```

Comment:
Please add this parameter in the readme

Reply:
https://github.com/percona/percona-helm-charts/pull/366/commits/7dd7151278231e4499304e488f95e51f3e0b1cad